### PR TITLE
Reformat the ML Config

### PR DESCRIPTION
**Pull request by tztsai**
I use a Python file to replace the `MLacc.def` config in txt format, so that the program can directly import the file by `from DEF_Trunk import config` and read the config variables as its attributes, instead of reading the lines of the txt file and parse the variables.

Closes part of #41

### CODE REVIEW

File: DEF_Trunk/config.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+"""Configuration file for the MLacc tasks."""
+
+logfile = "log.MLacc_Trunk"
+tasks = [
+    2,
+    4,
+]  # 1=test clustering, 2=clustering, 3=compress forcing, 4=ML, 5=evaluation
```
Comment (TomMelt) [inline DEF_Trunk/config.py:7]: Thanks for including comment to explain tasks

**Review by TomMelt — APPROVED** [review]
Looks good to me.

**Review by CALIPSO-LSCE — APPROVED** [review]
Tested the changes and packdata is produced, however it is only when we set `start_from_scratch = False` that packdata is produced and when set to True the code do not find 'packdata.nc'. Seems reverse. But we can adress it in another issue. Otherwise changes looks good.

**Comment by ma595** [comment]
> Tested the changes and packdata is produced, however it is only when we set `start_from_scratch = False` that packdata is produced and when set to True the code do not find 'packdata.nc'. Seems reverse. But we can adress it in another issue. Otherwise changes looks good.

This is a good point, behaviour should be reversed.

**Pull request merged**